.github/workflows/builds.yml

```diff
@@ -39,6 +36,14 @@ jobs:
         cd shacl
         python generate_shacl.py
         cd ..
+    - name: upload ttls
+      if: ${{ matrix.python-version }} == "3.7.9"
+      uses: actions/upload-artifact@v3
+      with:
+        name: ontology
+        path: |
+          Brick.ttl
+          shacl/BrickShape.ttl
```

Comment:
we don't need to include the `BrickShape.ttl` file ; it should be included in the `Brick.ttl` build.